docs/source/release_notes/release-notes.rst

```diff
@@ -41,6 +41,10 @@ develop
     *    - Type
          - Change
 
+    *    - |improved|
+         - Added Javadocs to CassandraKeyValueService.java, updated documentation.
```

Comment:
Could link to the file in GitHub here! "Update documentation" is a bit vague, I would say "Documented the behaviour of `CassandraKeyValueService` when one or more nodes in the Cassandra cluster are down", and not bother with the distinction between Javadocs and documentation. 

Are there supposed to be Java docs in this PR?